### PR TITLE
[TE]Yyuen/more_ember_addons

### PR DESCRIPTION
**Pull request by justYves**
### What's new:
- replaced `ember-networks` with `ember-fetch`
- Added 3 template helpers (`compute-color`, `compute-text-color` and `displayDate`)
- Added `ember-cli-sass`, change css to scss and created a wrapper folder
- Added ember-c3, ember-d3 (+shim).
- Added a `bower install` step to the maven build
- Added `ember-cli-spinjs` 
- Enabled source map for dev env

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/helpers/compute-color.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Template helper that computes the background color
+ * for the contribution map
+ * @param {Number} [value=0] Contribution change in percentage
+ * @return {String}          rgba color for background 
+ */
+export function computeColor([value = 0]) {
+  const opacity = Math.abs(value / 25);
+
+  if (value > 0){
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/helpers/compute-color.js]: Looks good - just an extra space preferred before the "{"

File: thirdeye/thirdeye-frontend/bower.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "name": "thirdeye-frontend",
   "dependencies": {
+    "d3": "^3.5.17",
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/bower.json]: Awesome. The fun begins!

File: thirdeye/thirdeye-frontend/app/helpers/compute-text-color.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+/**
+ * Template helper that computes the background color
```
Comment (yee-lkd) [inline thirdeye/thirdeye-frontend/app/helpers/compute-text-color.js]: Should this be "computes text color" instead (accordingly to file name)?

File: thirdeye/thirdeye-frontend/app/helpers/compute-text-color.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ * for the contribution map
+ * @param {Number} [value=0] Contribution change in percentage
+ * @return {String}          Text color (HEX)
+ */
+export function computeTextColor([value = 0]) {
+  const opacity = Math.abs(value/25);
+  if (opacity < 0.5) {
```
Comment (yee-lkd) [inline thirdeye/thirdeye-frontend/app/helpers/compute-text-color.js]: Nitpick - please add a line between declaration and the rest of the function.

File: thirdeye/thirdeye-frontend/app/helpers/format-date.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+/**
+ * Template Helper that displays human readable date
+ * @param {string} date A date
+ * @param {string} granularity granularity of the metric
+ * @return {string} human readable date
+ */
+export function formatDate([date, granularity='DAYS']) {
```
Comment (yee-lkd) [inline thirdeye/thirdeye-frontend/app/helpers/format-date.js]: Why not use "ember-moment" addon https://github.com/stefanpenner/ember-moment?

Comment (justYves) [inline thirdeye/thirdeye-frontend/app/helpers/format-date.js]: Ha nice found. Will definitely import!

File: thirdeye/thirdeye-frontend/app/helpers/compute-text-color.js (hunk 3 above)
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/helpers/compute-text-color.js]: Good catch!

**Comment by justYves** [comment]
@yee-lkd imported `ember-moment` and deleted the format date helper. 

Thanks!

**Pull request merged**